crates/bevy_sprite/src/render/mod.rs

```diff
@@ -337,7 +338,7 @@ pub fn prepare_sprites(
     let mut current_batch_colored = false;
     let mut last_z = 0.0;
     for extracted_sprite in extracted_sprites.sprites.iter() {
-        let colored = extracted_sprite.color != Color::WHITE;
+        let colored = extracted_sprite.color != 0xFFFFFFFF;
```

Comment:
Losing the nice name here is unfortunate. Ideally we could store at least this value as a named constant (although it is very recognizable).